DetourCrowd/Source/DetourObstacleAvoidance.cpp

```diff
@@ -331,7 +331,7 @@ float dtObstacleAvoidanceQuery::processSample(const float* vcand, const float cs
 	// find the threshold hit time to bail out based on the early out penalty
 	// (see how the penalty is calculated below to understnad)
 	float minPen = minPenalty - vpen - vcpen;
-	float tThresold = ((double)m_params.weightToi/(double)minPen - 0.1) * (double)m_params.horizTime;
+	float tThresold = (m_params.weightToi / minPen - 0.1f) * m_params.horizTime;
```

Comment:
Nice article!  I definitely learned a bit about FLOPs.

It looks like 99d1e791ab9898af18eb08b1e5a7983c5065143e introduced this.  Maybe @axelrodR can help clarify the intent here?